ethereum/contracts/ERC20App.sol

```diff
@@ -94,17 +96,19 @@ contract ERC20App {
         emit Unlocked(_token, _sender, _recipient, _amount);
     }
 
-    function encodeCall(address _token, address _sender, bytes32 _recipient, uint256 _amount)
-        private
-        pure
-        returns (bytes memory)
-    {
+    // SCALE-encode payload
+    function encodeCall(
+        address _token,
+        address _sender,
+        bytes32 _recipient,
+        uint256 _amount
+    ) private pure returns (bytes memory) {
         return
             abi.encodePacked(
                 MINT_CALL,
                 _token,
                 _sender,
-                byte(0x00), // Encode recipient as MultiAddress::Id
+                bytes1(0x00), // Encode recipient as MultiAddress::Id
```

Comment:
does this do anything different?

Reply:
looks like it would make it a byte array of length 1, rather than a single byte?

Reply:
I copied in the changes from https://github.com/Snowfork/polkadot-ethereum/commit/9c55d78da5727e87c322f49a42661e368087adb9 which has it as `bytes(0x00`), so not sure how it ended up as `bytes1`. Reverted to `bytes` in https://github.com/Snowfork/polkadot-ethereum/pull/266/commits/3c8835e6cc412c8c31fdcc7b62ca716c5a14d72d.

Reply:
Can we leave it as `byte`? I made it `byte` since according to the solidity docs, `byte` is an alias for `bytes1` (There isn't a single byte type in Solidity, as a fixed-length byte array of length 1 is equal to a single byte).

Reply:
Looks like my Solidity linter is automatically changing `byte` to `bytes1` on save, which is probably why this was originally changed to `bytes1`

Reply:
Changed it back to `byte` in https://github.com/Snowfork/polkadot-ethereum/pull/266/commits/7ef0b553f98b69069fb5b4f00f21f9fe24b147c7 :)